infrastructure/service-bus.tf

```diff
@@ -33,3 +33,32 @@ resource "azurerm_key_vault_secret" "sscs-hmc-servicebus-hared-access-key" {
   value        = data.azurerm_key_vault_secret.hmc-servicebus-shared-access-key.value
   key_vault_id = data.azurerm_key_vault.sscs_key_vault.id
 }
+  
+data "azurerm_servicebus_namespace" "servicebus_subscription_rule_sscs" {
+  name                = servicebus-subscription.namespace_name
+  resource_group_name = servicebus-subscription.resource_group_name
```

Comment:
These should be the same as the values given in the module before
https://github.com/hmcts/sscs-hearings-api/blob/5b31da26fc763a39ab8ed04de3217c8be03accf1/infrastructure/service-bus.tf#L5